docs/application/dotnet/get-started/first-app.md

```diff
@@ -0,0 +1,495 @@
+<style>
+    .tabcontent img {
+        border: 1px solid #555;
+        max-width: 100% !important;
+        max-height: 100%;
+    }
+</style>
+
+# Create your first Tizen .NET application
+
+The Tizen .NET framework allows you to easily and efficiently create applications for Tizen. Study the following instructions to help familiarize yourself with the Tizen .NET application development process. With the instructions, you can create and run a basic .NET application, which displays some text on the screen with no user interaction.
+
+1.  Before you get started with developing Tizen applications, set up the [development environment](../../../vstools/install.md).
```

Comment:
The link for **_development environment_** is broken here, please fix this.

Also, since dotnet/get-started/nui/first-app.md topic is removed now, we need to update text in dotnet/guides/user-interface/nui/text.md file referring to this first-app file. Please see the following text that is referring to first-app file.

"For more information on how to display a text using a text label, see NUI Hello World Tutorial."

Reply:
@mihashco the following part from the last review comment was not addressed: _since dotnet/get-started/nui/first-app.md topic is removed now, we need to update text in dotnet/guides/user-interface/nui/text.md file referring to this first-app file. Please see the following text that is referring to first-app file.
"For more information on how to display a text using a text label, see NUI Hello World Tutorial."_

Please address that and include the text.md file in this PR so we may review it.